Dockerfile-test

```diff
@@ -2,7 +2,7 @@ FROM python:2.7.13
 
 RUN curl -sL https://deb.nodesource.com/setup_8.x | bash - \
 	&& apt-get install -y nodejs
-RUN npm install -g solium
+RUN npm install -g AugurProject/solium
```

Comment:
@MicahZoltu VSCode uses the one from its `package.json`; which is `solium` right now; happy to change it though... not sure how to make it configurable since it's packaged with the extension